### ISSUE TITLE
Add term non-pitting edema

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10335]
**Preferred term label:**
Non-pitting edema

**Synonyms**
nonpitting edema, non-pitting oedema, nonpitting oedema

**Definition (free text, please give PubMed ID)**
Non-pitting edema occurs when excess fluid builds up in the body causing swelling that does not indent when pressure is applied. It usually occurs in the limbs, and often results from underlying medical conditions affecting lymphatic system function.
PMID: 32119339

**Parent term (use hpo.jax.org/app)**
Edema HP:0000969

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM275000, Graves disease
As example: PMID37352860

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000507
- label: Non-pitting edema
- parents: [Edema (HP:0000969)](https://hpo.jax.org/app/browse/term/HP:0000969)
- pmid: PMID:32119339
- def: Non-pitting edema occurs when excess fluid builds up in the body causing swelling that does not indent when pressure is applied. It usually occurs in the limbs, and often results from underlying medical conditions affecting lymphatic system function.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10335
- synonyms:
